src/Sulu/Bundle/MediaBundle/Media/FormatLoader/XmlFormatLoader.php

```diff
@@ -92,7 +92,13 @@ private function parseXml($file)
                     $parameters = array();
                     $parameterNodes = $this->xpath->query('x:parameters/x:parameter', $commandNode);
                     foreach ($parameterNodes as $parameterNode) {
-                        $parameters[$parameterNode->attributes->getNamedItem('name')->nodeValue] = $parameterNode->nodeValue;
+                        $value = $parameterNode->nodeValue;
+                        if ($value === 'true') {
+                            $value = true;
+                        } elseif ($value === 'false') {
+                            $value = false;
+                        }
+                        $parameters[$parameterNode->attributes->getNamedItem('name')->nodeValue] = $value;
```

Comment:
What's with all the other values not covered by your if statement before? And if it is only allowed to be `'true'` or `'false'` something like the following would be sufficient:

```php
$value = $parameterNode->nodeValue === 'true';
```